extensions/wikia/PhalanxII/PhalanxSpecialController.class.php

```diff
@@ -132,7 +132,6 @@ public function main() {
 		}
 
 		$this->setVal( 'expiries', $expiries );
-		$this->setVal( 'languages', $this->wg->PhalanxSupportedLanguages );
```

Comment:
actually it is used in rebuildPhalanxCache.php, I'll check if it can be removed there